api-report/core-interfaces.api.md

```diff
@@ -201,6 +253,14 @@ export interface ITaggedTelemetryPropertyType {
     value: TelemetryEventPropertyType;
 }
 
+// @public
+export interface ITaggedTelemetryPropertyTypeExt {
```

Comment:
i don't think this type should move here. it should stay in telemetry utils and should not be used in our public APIs

Reply:
I think this for all the ext types

Reply:
curious what @markfields  thinks

Reply:
I'll investigate options.

Reply:
Is the problem that these are used by the stuff you're trying to move?  Unfortunately I'm about to go OOF, but can check back when I'm back in 2 weeks.  Regardless of where it lives, only `ITelemetryBaseLogger` and the types it needs should be used on any public (non-internal, if that's even well-defined) API or across compat boundaries:

```
ITelemetryBaseLogger
ITelemetryBaseEvent
ITelemetryProperties
ITaggedTelemetryPropertyType
TelemetryEventPropertyType
```

So even `ITelemetryLogger` and related types which are currently in the same file with those should be made distinct in some way (actually all those should be deprecated in favor of the `Ext` versions here... I thought they were deprecated but apparently not anymore, and more usages are popping up!)

Reply:
See AB#4545

Reply:
And as you're dealing with error types too, AB#2802 may also be relevant. Not saying you should fix it, but just for context on the direction. Which I'd summarize as:

* Every instrumentation point in our code should have access to the Ext versions, to support logging arrays/objects (Objects are esp important to reduce number of columns in a telemetry DB like Kusto)
* Most APIs should pass around a Base logger and consumer should pass that to `createChildLogger` to get the Ext thing.